frontend/src/i18n/fr/translation.json

```
@@ -543,7 +544,7 @@
     "total": "Totale",
     "general": "Général",
     "other": "Autre",
-    "no_data":" Pas de données"
+    "no_data": " Pas de données"
```

```suggestion
    "no_data": "Pas de données"
```
Hi @kartik1112, can you please remove the extra space in the beginning of this value.